### DISCUSSION TITLE
Deploy On Railway!

### DISCUSSION THREAD

**Discussion by kaansenol** [devlikeapro/waha#700] [Q&A]
Hi everyone,

I'm having trouble deploying Waha-Plus on the Railway (railway dot com). Has anyone here successfully deployed Waha-Plus on Railway and can share their experience? Alternatively, I'd love to know where most of you are hosting Waha online.

Thanks in advance!

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Here's the guide to install using docker-compose
https://waha.devlike.pro/docs/how-to/install/
Try following it and set up the right env variables in railway

Or consider getting any VPS with 2CPU/2 Memory minimum (check requirements based on the engine and # of session you're going to run https://waha.devlike.pro/docs/overview/faq/#system-requirements)
Something like https://pq.hosting/en/

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by lucaskafeapps** [comment]
hey buddy, i am having the same issue here, can you share if you have sucess?

**Reply by kaansenol** [reply]
I couldn't manage to deploy on Railway, so I switched to DigitalOcean.
